### PR TITLE
readme: add notes on per runs

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -28,6 +28,22 @@ launchChromeAndRunLighthouse('https://example.com', flags).then(results => {
 });
 ```
 
+## Performance-only Lighthouse run
+
+Many modules consuming Lighthouse are only interested in the performance numbers. 
+Lighthouse ships with a [performance-only config](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/perf.json) that you can use:
+
+```js
+const perfConfig: any = require('lighthouse/lighthouse-core/config/perf.json');
```
Comment (ebidel) [inline docs/readme.md]: `launchChromeAndRunLighthouse` to be consistent with the other example
```javascript
const perfConfig = require('lighthouse/lighthouse-core/config/perf.json');`
// ...
launchChromeAndRunLighthouse(url, flags, perfConfig).then(results => {...});
```

File: docs/readme.md [hunk 2]
```diff
@@ -28,6 +28,22 @@ launchChromeAndRunLighthouse('https://example.com', flags).then(results => {
 });
 ```
 
+## Performance-only Lighthouse run
```
Comment (ebidel) [inline docs/readme.md]: use h3: `###`

File: docs/readme.md [hunk 3]
```diff
@@ -28,6 +28,22 @@ launchChromeAndRunLighthouse('https://example.com', flags).then(results => {
...
+
+```js
+const perfConfig: any = require('lighthouse/lighthouse-core/config/perf.json');
+
+// ...
+
+lighthouse(url, flags, perfConfig)
+```
+
+You can also craft your own config (e.g. [plots.json](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/plots.json)) for completely custom runs.
```
Comment (ebidel) [inline docs/readme.md]: https://github.com/GoogleChrome/lighthouse/tree/master/docs/recipes/custom-audit#the-audit-gatherer-and-config might be a better link.

**Comment by paulirish** [comment]
sg. done.

**Pull request merged**